api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -527,12 +510,9 @@ function networkAccountReducer(address, account, store) {
     }
   }
   return {
-    name:
-      account && account.identity && account.identity.display
-        ? account.identity.display
-        : '',
+    name: account.value ? hexToString(account.value.info.display.Raw) : '',
     address,
-    picture: account ? account.twitter : ''
+    picture: '' // TODO: we need to get images from twitter account
```

Comment:
can this be done now?

Reply:
Yep, but I would open a different PR for that. Claim rewards notification depends on this to be completed.